load-test/walkthrough-documents/Create-Full-Production-Size-Static-Test-Data-for-Load-Testing-v01-draft.md

```diff
@@ -0,0 +1,61 @@
+# Create Full Production Size Static Test Data for Load Testiing
+---
+
+## Task Overview
+
+- Create a Teacher for each School
+
+
+## Background
+
+
+
+## Create a Teacher for each School
+
+
+
+```sql
+
+--==============================================================
+-- UNIQUE CODE BLOCK IDENTIFIER - STA-MTC-LOAD-TEST-CODE-001v001
+
+-- CODE VERSION v001
+
+
+-- CREATE ONE TEACHER FOR EACH SCHOOL -- START SQL CODE BLOCK
```

Comment:
I think it would be better to use the `school` table to drive the `user.school_id` population, and I am not entirely sure what the `loopFrom` and `loopTo` are for.

For me, it would be simpler to write something like:

```SQL
INSERT INTO [mtc_admin].[user] (
	identifier, passwordHash, school_id,
	role_id
)
SELECT
	'Teachernew' + CAST(
		s.id AS VARCHAR(6)
	),
	'$2a$10$.WsawgZpWSAQVaa6Vz3P1.XO.1YntYJLd6Da5lrXCAkVxhhLpkOHK',
	s.id,
	(select id from mtc_admin.role where title = 'TEACHER')
FROM
	mtc_admin.school s;
```

Load time is excellent:

```
Initializing connection...
Connection ready. Took 0.408 seconds.

Statement completed successfully. (16841 rows affected). 0.000 seconds. (Line 1).

Finished. Total time: 0.408 seconds.
```